update Node compatibility to be more explicit about version support (#4625)
DataDog/documentation: content/en/tracing/setup/nodejs.md

## Compatibility

> Node `>=4` is supported by this library. We recommend always using the latest LTS to avoid known issues and performance degradation. Odd version numbers are not officially supported.

Reviewer: ```suggestion
Only Node `>=4` is supported by this library but you should use the latest LTS to avoid known issues and performance degradation. 
```
